### ISSUE TITLE
LC0040 Wrong Warning for Insert on MediaSet

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#406]
The Insert method on a MediaSet Field shouldn't trigger the Warning LC0040
```al
Item.Picture.Insert(MediaId);
```

See: https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/mediaset/mediaset-insert-method

**Comment by Arthurvdv** [comment]
Thanks, I've also encountered a false warning on  ```IsolatedStorage.Delete()``` in the meantime myself.

I've now bound the rule to record types and these false positives should now be resolved in the (pre)release version of [v0.30.9](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.9).

**Comment by Arthurvdv** [comment]
@fvet, valid question! It did not include RecordRef's 🙈

I've now bound the rule to Record ánd RecordRef types in the (pre)release version of [v0.30.9](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.9).
